app/models/administrative-unit.ts

```diff
@@ -1,13 +1,15 @@
 import Model, {
+  AsyncBelongsTo,
+  AsyncHasMany,
   attr,
   belongsTo,
   hasMany,
-  AsyncHasMany,
-  AsyncBelongsTo,
 } from '@ember-data/model';
 import GoverningBodyModel from './governing-body';
-import AdministrativeUnitClasssificationCodeModel from './location';
-import LocationModel from './location';
+import {
+  default as AdministrativeUnitClasssificationCodeModel,
```

Comment:
`AdministrativeUnitClasssificationCodeModel` path is not correct, it should be `./administrative-unit-classification-code` instead of `./location'.